__tests__/test.test.js

```diff
@@ -113,5 +113,48 @@ describe('GET /api/articles/:article_id', () => {
         expect(body.msg).toEqual('Bad request')
         })
     })
-    })
+ })
 
+describe('GET /api/articles', () => {
+    test('Status code 200 returned', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        })
+
+    test('Request responds with an article that has the correct properties and comment count', () => {
+        return request(app)
+        .get('/api/articles')
+        .then(({ body }) => {
+            expect(body.articles[6]).toMatchObject({
```

Comment:
You should check each article matches rather just one. Consider using a forEach.